assets/tasks/common.json

```diff
@@ -1017,6 +1017,35 @@
                 "id": "flame",
                 "color": "terminal.ansiYellow"
             }
+        },
+        {
+            "label": "show-project-documentation",
+            "command": "${command:markdown.showPreview}",
+            "type": "process",
+            "dependsOn": [
+                "open-project-documentation-file"
+            ],
+            "dependsOrder": "sequence",
+            "problemMatcher": "$msCompile",
+            "icon": {
```

Comment:
add here:

```json
"presentation": {
                "echo": false,
                "reveal": "never",
                "focus": false,
                "panel": "shared",
                "showReuseMessage": false,
                "clear": true
            },
```

so the output panel is not presented and focused, catching the attention ...